packages/framer-motion/src/value/scroll/use-viewport-scroll.ts

```diff
@@ -21,7 +21,6 @@ let hasListeners = false
 
 function addEventListeners() {
     hasListeners = true
-    if (typeof window === "undefined") return
```

Comment:
`addEventListeners` is called in an effect, which makes this check redudant. But let me know if you feel it's better to leave it in.